pom.xml

```diff
@@ -1261,6 +1268,18 @@
                 </exclusions>
             </dependency>
 
+            <dependency>
+                <groupId>org.apache.kafka</groupId>
+                <artifactId>kafka-clients</artifactId>
+                <version>${kafka.version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>org.slf4j</groupId>
+                        <artifactId>slf4j-api</artifactId>
+                    </exclusion>
+                </exclusions>
```

Comment:
unnecessary exclusion

Reply:
why we should skip this exclusion? `kafka-clients` is depending on `slf4j-api`.

Reply:
We should respect the Maven transitive dependencies in most cases, just excluding the deps we truly don't need, and overriding versions in `dependencyManagement`(just overriding, without excluding)

But it's not suitable for components which is disastrous for dependencies management, like Hive, Hudi.

Reply:
SGTM. Removed this exclusion.